crates/autopilot/src/domain/competition/winner_selection/combinatorial.rs

```diff
@@ -97,6 +97,14 @@ impl Arbitrator for Config {
             .into_iter()
             .enumerate()
             .map(|(index, participant)| participant.rank(winner_indexes.contains(&index)))
+            .sorted_by_key(|p| {
+                (
+                    // winners before losers
```

Comment:
nit: not sure if it is a proper term here
```suggestion
                    // winners before non_winners
```